packages/lambdas/src/sqs-to-converter.ts

```diff
@@ -427,6 +427,16 @@ export const handler = Sentry.AWSLambda.wrapHandler(async (event: SQSEvent) => {
           timestamp: new Date(),
         };
 
+        if (!payloadRaw.trim().length) {
+          capture.message("XML document is empty", {
+            extra: { context: lambdaName, fileName: s3FileName },
```

Comment:
I suggest adding additional info to `extra`, like patientId, cxId, docId, requestId